### ISSUE TITLE
Feature request: add undo

### ISSUE THREAD

**Issue by Gys** [andydotxyz/fynebuilder#18]
Very interesting idea to have a visual builder. I was toying with it a little but quickly ran into a problem that I maximized something and had no way to go back. Having an undo (maybe multiple steps even) would be very useful at that point, so the user can backtrack and continue working. Now I had to quit the app.

**Comment by andydotxyz** [comment]
Agreed. We will need to code it in manually as Fyne has not got built-in undo/redo yet.
